src/uwtools/resources/FV3Forecast.jsonschema

```diff
@@ -3,116 +3,71 @@
     "description": "This document is to validate user-defined FV3 forecast config files",
     "type": "object",
     "properties": {
-        "resources": {
+        "platform": {
             "description": "Expected resources for the given platform",
             "type": "object",
             "properties": {
-                "WORKFLOW_MANAGER": {
+                "RUN_ENVIR": {
                     "type": "string",
+                },
+                "MACHINE": {
+                    "type": "string"
                     "enum": [
-                        "rocoto",
-                        "none"
+                        "cheyenne",
+                        "gaea",
+                        "hera",
+                        "jet",
+                        "noaacloud",
+                        "orion",
+                        "wcoss2"
                     ]
                 },
-                "NCORES_PER_NODE": {
-                    "type": "number"
-                },
-                "SCHED": {
+                "MPICMD": {
                     "type": "string",
                     "enum": [
-                        "slurm",
-                        "pbspro",
-                        "lsf",
-                        "lsfcray",
-                        "none"
+                        "mpirun",
+                        "mpiexec",
+                        "srun"
                     ]
+                },
+                "ACCOUNT": {
+                    "type": "string"
                 }
             }
         },
-        "parameters": {
+    "forecast": {
             "description": "parameters of the forecast",
             "type": "object",
             "properties": {
-                "LENGTH": {
+                "APP": {
                     "type": "string",
-                    "format": "number"
                 },
-                "GRID": {
+                "EXTRN_MDL_NAME": {
```

Comment:
This one is not needed.

Reply:
We had previously intended to move the model name to the YAML, do we want to revert that back to the CLI?

Reply:
Would we have access to this fact from the CLI, e.g. because we'd run `uw forecast run --model=fv3` or similar? Certainly by the time we've instantiated an `FV3Forecast` object we know that we're running FV3, but would we want that information to come from CLI or YAML? (Not both, I'd probably argue.)

Reply:
Oh. I see. In SRW `EXTRN_MDL_NAME` is the variable that defines which system to get initial and boundary conditions from. We do not want to use this variable if what we are after is `model` to mean that we are looking at fv3, mpas, etc. Maybe let's go with `model` as in @maddenp-noaa's example. It can live here as we'd discussed previously.